assets/css/style.css

```diff
@@ -0,0 +1,1262 @@
+@import url("../fonts/Lato-Light.ttf");
+@import url("../fonts/Cocogoose Pro Semilight-trial.ttf");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+  overflow-x: hidden;
+}
+
+body {
+  overflow-x: hidden;
+}
+
+.btn-primary:hover {
+  background-color: #fff;
+  border: 1px solid #ec5242;
+  color: #ec5242;
+}
+
+.card:hover {
+  border: 2px solid #fff;
+  cursor: pointer;
+}
+
+.nav-item a:hover {
+  cursor: pointer;
+  color: #ec5242;
+}
+
+#mlist li a:hover,
+.close:hover {
+  cursor: pointer;
+  color: #ec4542;
+}
+
+@media (max-width: 767px) {
+  .header {
+    background: url(../images/bg-image.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    box-shadow: inset 0 0 0 1000px rgba(210, 210, 209, 0.8);
+  }
+
+  .hamburger-icon {
+    display: inline-block;
+    cursor: pointer;
+    margin-top: 10px;
+    margin-left: 30px;
+  }
+
+  .bar1,
+  .bar2,
+  .bar3 {
+    width: 25px;
+    height: 3px;
+    background-color: rgb(78, 78, 78);
+    margin: 6px 0;
+    transition: 0.4s;
+  }
+
+  .hero-paragraph {
+    color: #ec5242;
+    font-family: "Lato", sans-serif;
+    font-weight: 500;
+    line-height: 2rem;
+    margin: 0 0 0 1rem;
+    margin-top: 30px;
+    font-size: 15px;
+  }
+
+  .hero-heading {
+    color: #ec5242;
+    font-family: "Cocogoose", sans-serif;
+    width: 250px;
+    font-size: 22px;
+    text-transform: uppercase;
+    margin: 0 0 0 1rem;
+    font-weight: bold;
+  }
+
+  .hero-description {
+    border: 2px solid #f4f4f4;
+    background-color: #d3d3d3;
+    margin: 0.3rem 0 0.3rem 1rem;
+    width: 55rem;
+    max-width: calc(100% - 2rem);
+    margin-top: 30px;
+  }
+
+  .hero-description p {
+    font-family: "Lato", sans-serif;
+    line-height: 1.5rem;
+    margin: 1rem;
+  }
+
+  .hero-date {
+    font-family: "Cocogoose", sans-serif;
```

Comment:
## optional:
- [ ] For a DRY code, please remove this line occuring multiple times and use once for grouping the selectors or adding it to the universal selector.